docs/install.md

```diff
@@ -133,11 +133,21 @@ To **generate a new Guld identity** and PGP key:
 
 To **import an existing Guld identity** and PGP key:
 
-> **TODO** it looks like the UI for this one is broken right now. Choosing import doesn't change the UI at all.
-
 1. Select the "import" radio button.
 
-1. ...
+1. Export the private key you used to register your Guld Identity from your preferred PGP key manager
+
+1. Copy and paste the exported private key into the text area labeled ASCII Armored Private Key
+
+1. Enter your existing Guld username
+
+1. Enter a strong passphrase for the private PGP key, and confirm it.
+
+    Keep your passphrase stored securely! There is no way to recover a lost passphrase.
```

Comment:
Delete this note. They should know already they need to be careful with an existing key, right?

Reply:
I would assume so but keeping this in helps us to not be held accountable by someone saying "well you didn't tell me it wasn't backed up", password recovery is a very common feature in centralized applications so someone may be under the impression it's stored somewhere

Reply:
That's legit... leave it in!

Reply:
good point!